scss/_buttons.scss

```diff
@@ -1,51 +1,97 @@
+// stylelint-disable custom-property-empty-line-before
+
 //
 // Base styles
 //
 
 .btn {
+  // scss-docs-start btn-css-vars
+  --#{$variable-prefix}btn-padding: #{$btn-padding-y} #{$btn-padding-x};
```

Comment:
Couldn't we add separate custom props for vertical and horizontal padding? I guess this request will pop up for each component.

May even drop `--bs-*-padding` since they're only shorthands, don't you think?